src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/notify.js

```diff
@@ -1,20 +1,35 @@
 const Transaction = require('dw/system/Transaction');
 const checkAuth = require('*/cartridge/scripts/checkNotificationAuth');
 const handleNotify = require('*/cartridge/scripts/handleNotify');
+const AdyenConfigs = require('*/cartridge/scripts/util/adyenConfigs');
 
 /**
  * Called by Adyen to update status of payments. It should always display [accepted] when finished.
  */
+
+function handleHmacVerification(req){
```

Comment:
This function might return undefined instead of boolean. 
Instead maybe you can use
```
if (!hmacKey) {
return false;
}
return checkAuth.validateHmacSignature(req);
```